packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -332,7 +330,7 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
       _pixels != null,
       'An initial pixels value must exist by calling correctPixels on the ScrollPosition',
     );
-    _pixels += correction;
+    _pixels = pixels! + correction;
```

Comment:
use _pixels here not pixels, in case pixels is overriden; but, in theory, this should simplify back to the old code

Reply:
I replaces `pixels` with `_pixels` but I cannot go back to old code because `_pixels` is nullable.

Reply:
yeah if we must make _pixels nullable then fair enough.